kratos/processes/find_intersected_geometrical_objects_process.h

```diff
@@ -7,355 +7,667 @@
-
-
-      /// Destructor.
-      ~FindIntersectedGeometricalObjectsProcess() override {}
-
-      ///@name Member Variables
-      ///@{
-
-      std::vector<PointerVector<GeometricalObject>> mIntersectedObjects;
-
-      ///@}
-      ///@name Operations
-      ///@{
-
-      virtual void Initialize();
-
-      virtual void FindIntersectedSkinObjects(std::vector<PointerVector<GeometricalObject>>& rResults);
-
-      virtual void FindIntersections();
-
-      virtual std::vector<PointerVector<GeometricalObject>>& GetIntersections();
-
-      virtual ModelPart& GetModelPart1();
-
-      virtual OctreeBinary<OctreeBinaryCell<Internals::DistanceSpatialContainersConfigure>>* GetOctreePointer();
-
-      virtual void Clear();
+            KRATOS_ERROR << "Not Implemented method" << std::endl;
+//             return HasIntersection(pObject->GetGeometry(), low_point, high_point);
+        }
 
-      void Execute() override;
+        ///@}
+        ///@name Input and output
+        ///@{
 
-      ///@}
-      ///@name Input and output
-      ///@{
+        /// Turn back information as a string.
+        virtual std::string Info() const
+        {
+            return " Spatial Containers Configure";
+        }
 
-      /// Turn back information as a string.
-      std::string Info() const override;
+        /// Print information about this object.
+        virtual void PrintInfo(std::ostream& rOStream) const {}
 
-      /// Print information about this object.
-      void PrintInfo(std::ostream& rOStream) const override;
+        /// Print object's data.
+        virtual void PrintData(std::ostream& rOStream) const {}
 
-      /// Print object's data.
-      void PrintData(std::ostream& rOStream) const override;
+        ///@}
 
-      ///@}
+    protected:
 
     private:
-      ///@name Static Member Variables
-      ///@{
-
-
-      ///@}
-      ///@name Member Variables
-      ///@{
-
-        ModelPart& mrModelPart1;
-        ModelPart& mrModelPart2;
-        OctreeType mOctree;
-
-      ///@}
-      ///@name Private Operations
-      ///@{
 
-        void GenerateOctree();
-        void SetOctreeBoundingBox();
-        void MarkIfIntersected(Element& rElement1, std::vector<OctreeType::cell_type*>& leaves);
-        bool HasIntersection(Element::GeometryType& rFirstGeometry, Element::GeometryType& rSecondGeometry);
-        bool HasIntersection2D(Element::GeometryType& rFirstGeometry, Element::GeometryType& rSecondGeometry);
-        bool HasIntersection3D(Element::GeometryType& rFirstGeometry, Element::GeometryType& rSecondGeometry);
-        void FindIntersectedSkinObjects(Element& rElement1, std::vector<OctreeType::cell_type*>& leaves, PointerVector<GeometricalObject>& rResults);
+        /// Assignment operator.
+        DistanceSpatialContainersConfigure& operator=(DistanceSpatialContainersConfigure const& rOther);
 
+        /// Copy constructor.
+        DistanceSpatialContainersConfigure(DistanceSpatialContainersConfigure const& rOther);
```

Comment:
```suggestion
        DistanceSpatialContainersConfigure(DistanceSpatialContainersConfigure const& rOther) = delete;
```

Reply:
Is there any reason to avoid deleting the copy constructor?

Reply:
I updated manually